src/Set.sol

```diff
@@ -2,6 +2,10 @@
 
 pragma solidity ^0.8.19;
 
+/// @dev Represents the maximum number of elements that can be stored in the set.
+/// Must not exceed 255 due to the uint8 data type limit.
+uint8 constant MAX_ELEMENTS = 10;
```

Comment:
If this constant is getting globally exported, would it make sense to be `SET_MAX_ELEMENTS` or something?